package.json

```diff
@@ -368,7 +368,7 @@
     "@brave/brave-ui": "0.40.4",
     "@brave/leo": "github:brave/leo#857181f3d314e1c55bb2e19e21924acb0b54c587",
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
-    "@brave/wallet-standard-brave": "~0.0.11",
+    "@brave/wallet-standard-brave": "~0.0.12",
```

Comment:
any reason we are not doing exact version matching here?

Reply:
I believe it was put this way so we don't necessary need to bump this in core (and uplift to every channels) every time for minor changes.
I bump here this time because I think it's important enough which I want to make sure it will be used in any cases (such as local npm cache not cleared or so).

Reply:
Is exact version more preferred?

Reply:
it's not clear to me though how this interacts with package-lock.json... https://stackoverflow.com/questions/62560311/when-npm-install-ci-honors-package-lock-json-is-there-any-purpose-of-using-tild implies that it's not going to update the patch version anyway in core unless someone manually runs npm update or something.

in that case i don't think we are getting any benefit from using the tilde.

Reply:
Ugh, okay, in this case I'll update it to exact version, thanks.